.github/ISSUE_TEMPLATE/bug_report_form.yml

```diff
@@ -0,0 +1,130 @@
+              Please write the full details of your problem
+      validations:
+          required: true
+    - type: textarea
+      id: reproduce
+      attributes:
+          label: Steps to reproduce
+          description: |
+              Describe the steps to reproduce the bug.
+              The better your description is _(go 'here', click 'there'...)_ the fastest you'll get an _(accurate)_ answer.
+          value: |
+              1.
+              2.
+              3.
+      validations:
+          required: true
+    - type: textarea
+      id: expected-behavior
+      attributes:
+          label: Expected behavior
+          description: Describe what you expected to happen instead.
+      validations:
+          required: true
+    - type: textarea
+      id: mail-app-version
+      attributes:
+          label: Mail app version
+          description: |
+              See apps admin page, e.g. 0.5.3
+    - type: textarea
+      id: mail-server-service
+      attributes:
+          label: Mailserver or service
+          description: |
+              e.g. Outlook, Yahoo, Gmail, Exchange...
+    - type: dropdown
+      id: system
+      attributes:
+          label: Operating system
+          description: |
+              Select operating system where Nextcloud Mail app is installed.
+              _Describe in the "Additional info" section if you chose "Other"._
+          options:
+              - "Debian/Ubuntu"
+              - "RHEL/CentOS"
+              - "Other"
+    - type: dropdown
+      id: php
+      attributes:
+          label: PHP engine version
+          description: |
+              Select PHP engine version serving Nextcloud Server.
+              _Describe in the "Additional info" section if you chose "Other"._
+          options:
+              - "PHP 7.3"
+              - "PHP 7.4"
+              - "PHP 8.0"
+              - "PHP 8.1"
+              - "Other"
+    - type: dropdown
+      id: webserver
+      attributes:
+          label: Web server
+          description: |
+              Select Webserver serving Nextcloud Server.
+              _Describe in the "Additional info" section if you chose "Other"._
+          options:
+              - "Apache (supported)"
+              - "Nginx"
+              - "Lighttpd"
+              - "Other"
+    - type: dropdown
+      id: database
+      attributes:
+          label: Database engine version
+          description: |
+              Select Database engine serving Nextcloud Server.
+              _Describe in the "Additional info" section if you chose "Other"._
+          options:
+              - "MySQL"
+              - "MariaDB"
+              - "PostgreSQL"
+              - "SQlite"
+              - "Oracle"
+              - "Other"
+    - type: textarea
+      id: nc-server-version
+      attributes:
+          label: Nextcloud Version
+          description: |
+              See admin page, e.g. 22.0.2
+    - type: textarea
```

Comment:
See https://github.com/nextcloud/mail/issues/new?assignees=&labels=bug%2C+0.+to+triage&template=bug.md&title=. We don't ask for this.